documents/policies/external/Competition Requirements.md

```diff
@@ -15,25 +15,30 @@ An official WCA Competition must:
    1. Exception: in case the WCA Delegate cannot attend the competition, the WCA Board may appoint a trusted person to act as a temporary WCA Delegate for the competition.
    2. The WCA Delegate for a competition is responsible for:
       1. Submitting the Delegate Report to the WCA Regulations Committee and the WCA Quality Assurance Committee regarding adherence to WCA Regulations and quality standards during the competition, the overall course of the competition, and any incidents that happened. The report must be submitted within one week of the end of the competition.
       2. Submitting the competition results to the WCA Results Team within one week of the end of the competition.
-         1. Results should be double-checked and free of errors before submission.
+         1. Results must be double-checked and should be free of errors before submission.
          2. All scramble sequences used during a competition must be sent with the results of the competition.
             1. All scramble sequences must be labeled with the events, rounds, and groups for which they were used. If there are multiple groups in a round, it is not necessary to identify which competitor was in which group.
-         3. Physical records (e.g. score sheets, written solutions from 3x3x3 Fewest Moves) must be retained for a minimum of 90 days. Digital records (e.g. scramble sequences, results) must be retained for a minimum of 7 years.
+         3. Record keeping
+            1. Physical records (e.g. score sheets, written solutions from 3x3x3 Fewest Moves) must be retained for a minimum of 90 days.
+                 1. Scanned copies may be kept instead of the physical records.
+            2. Digital records (e.g. scramble sequences, results) must be retained for a minimum of 7 years.
       3. Sending corrections of competition results to the WCA Results Team.
       4. Advising other competition officials when necessary.
-      5. Deciding about disqualifying competitors during the competition.
+      5. Deciding whether to disqualify competitors during the competition per ([WCA Regulation 2k)](wca{regulations/#2k})).
       6. Generating, providing, and maintaining the security of the scramble sequences ([WCA Regulation 4b2)](wca{regulations/#4b2})).
       7. Ensuring that all outstanding incidents after a competition are resolved.
    3. The WCA Delegate may delegate some of their responsibilities to the organization team or other officials.
-9. Comply with the rules of “qualified sponsorship” set forth by Publication 598 of the Internal Revenue Service (IRS), as well as any additional rules or laws in the local country of the competition.
+9. The information listed on the WCA Website, as well as other official WCA channels must comply with the rules of “qualified sponsorship” set forth by Publication 598 of the Internal Revenue Service (IRS), as well as any additional rules or laws in the local country of the competition.
```

Comment:
But at the competition itself the rules don't need to be followed? I'm fine with this, but I'm still not sure that there isn't a general presentation that WCA competitions are somehow organised by the WCA, where qualified sponsorship could become an issue depending on how Organisers / Delegates (noting they may be the same person) present WCA involvement.

Reply:
This is a difficult point because we (WCAT) can't enforce how competitions comply with this point beyond the information on the competition website. You're absolutely right that we don't do well at showing the difference of the WCA versus organizers, but this is far out of my wheelhouse.